### ISSUE TITLE
WebP, AVIF, JPEG XL support

### ISSUE THREAD

**Issue by xplosionmind** [alexharris/ditherit-v2#40]
It would be great if both input and output could support other more efficient formats, too!

**Comment by alexharris** [comment]
Yep this would be good. Rigth now it looks like the method used to create a download, [toDataURL](https://developer.mozilla.org/en-US/docs/Web/API/HTMLCanvasElement/toDataURL) currently only supports jpg, png, and webp except for in Safari. Maybe there is a library that allows for other options.
